### DISCUSSION TITLE
Cant uninstall Vigilante, please help

### DISCUSSION THREAD

**Discussion by realitycrashexe** [FunkyMuse/Vigilante#316] [Q&A] [answered]
Does anyone have a solution that can uninstall this app? Ive tried uninstall from fdroid, from settings menu, force stops, clear cache, disabling, everything ive tried doesn't work, and package installer keeps stopping. From what I've read online pretty much everyone has this problem, given that, it just really seems like a very sketchy thing for an app not able to be uninstalled, from what I've read, on any device at all, just my opinion that it's sketchy/malicious if noone can uninstall this thing without doing a factory reset, which is total b.s honestly. 
Thanks in advance to whoever helps me out here

**Comment by FunkyMuse** [comment, marked as answer]
It's in the readme https://github.com/FunkyMuse/Vigilante/issues/71#issuecomment-769303018
